03-kubernetes&openshift/03.3-helm.md

```diff
@@ -0,0 +1,16 @@
+# Helm
+
+### TODO:
+
+1. Read about [helm](https://letmegooglethat.com/?q=helm) 
+
+
+### Key Concepts:
+- chart
+- release
+- repo 
+- jinja2
+- templates
```

Comment:
Add the concept of a subchart, template functions and kustomization file. Also helm uses go templates, not jinja2

Capital letters might be nice